example/notebook/installation_register_and_convert_model.ipynb

```diff
@@ -0,0 +1,481 @@
+     ]
+    }
+   ],
+   "source": [
+    "%%bash\n",
+    "modelci start"
+   ]
+  },
+  {
+   "source": [
+    "## 2. Register ResNet50 Model\n"
+   ],
+   "cell_type": "markdown",
+   "metadata": {}
+  },
+  {
+   "source": [
+    "Firstly, we load pre-trained resnet50 model from torchvision, you can refer to <https://pytorch.org/docs/stable/torchvision/models.html> for more examples of pretrained models."
+   ],
+   "cell_type": "markdown",
+   "metadata": {}
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# load resnet50 model from torchvision\n",
+    "from torchvision import models\n",
+    "model = models.resnet50(pretrained=True)"
+   ]
+  },
+  {
+   "source": [
+    "Then we register this model into ModelHub"
+   ],
+   "cell_type": "markdown",
+   "metadata": {}
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [
+    {
+     "output_type": "stream",
+     "name": "stderr",
+     "text": [
+      "2020-11-24 15:38:39,221 - converter - INFO - Use cached model\n",
+      "2020-11-24 15:38:39,224 - converter - INFO - Use cached model\n"
+     ]
+    }
+   ],
+   "source": [
+    "from modelci.hub.manager import register_model\n",
+    "from modelci.types.bo import Framework, IOShape, ModelVersion,Engine\n",
+    "from modelci.types.trtis_objects import ModelInputFormat\n",
+    "# set model input and output formats\n",
+    "inputs = [IOShape([-1, 3, 224, 224], dtype=float, name='INPUT__0', format=ModelInputFormat.FORMAT_NCHW)]\n",
+    "outputs = [IOShape([-1, 1000], dtype=float, name='probs')]\n",
+    "# register the model\n",
+    "register_model(\n",
+    "    model,\n",
+    "    dataset='imagenet',\n",
+    "    acc=0.76,\n",
+    "    task='image classification',\n",
+    "    inputs=inputs,\n",
+    "    outputs=outputs,\n",
+    "    architecture='ResNet50',\n",
+    "    framework=Framework.PYTORCH,\n",
+    "    version=ModelVersion('2')\n",
+    ")"
+   ]
+  },
+  {
+   "source": [
+    "## 3. Retrieve Models\n",
```

Comment:
keep all the titles/subtitles in the same letter style, all in upper case or not.